HACKING.md

```diff
@@ -24,6 +25,14 @@
 
 <!-- tocstop -->
 
+## Online
```

Comment:
I'm not OK with putting this _before_ the "building hotspot" section. I'm also lacking some kind of motivation - when would one use this setup? To actually work on hotspot and write patches without a local dev environment? Or just to run hotspot "in the browser"? Because building isn't needed thanks to appimages either?

Reply:
The answer is "to actually work on hotspot in the browser, writing patches in a clean environment, not needing a complete local dev environment", I've adjusted that and moved the Online part one below the building.